dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+ return type that is supported by Jakarta Data or is a subclass of Iterable with \
+ a public constructor that does not have parameters.
+
+CWWKD1058.no.args.constr.err=CWWKD1058E: The {0} return type of the {2} method \
+ of the {3} repository interface is not valid for a repository method because the \
+ constructor of the {4} class raised an error: {5}.
+CWWKD1058.no.args.constr.err.explanation=The public constructor without parameters \
+ failed with an error.
+CWWKD1058.no.args.constr.err.useraction=Correct the cause of the error, or update \
+ the repository method to have a different return type.
+
+CWWKD1059.prop.cast.err=CWWKD1059E: The {0} method of the {1} repository \
+ interface cannot obtain the {2} property of the {3} entity because the \
+ {4} field or method of the {5} class cannot be used on a {6} value.
+CWWKD1059.prop.cast.err.explanation=An entity property or a property from its \
+ hierarchy of relations is not compatible the expected type.
+CWWKD1059.prop.cast.err.useraction=Review the entity model and ensure that the \
+ expected types are used for each property.
+
+CWWKD1060.name.out.of.scope=CWWKD1060E: The {0} repository interface from the \
+ {1} application configures a {2} value for the {3} attribute, but {3} names \
+ are not accessible to the artifact that provides the repository interface. \
+ Use a {4} name instead.
+CWWKD1060.name.out.of.scope.explanation=The specified namespace is not visible \
+ to the artifact in which the repository interface is defined.
+CWWKD1060.name.out.of.scope.useraction=Update the Repository annotation to \
+ specify a data store name that is visible to the module or library that \
+ provides the repository interface.
+
+CWWKD1061.comp.name.in.ejb=CWWKD1061E: The {0} repository interface that is \
+ defined in the {1} enterprise bean module of the {2} application configures \
+ a {2} value for the {3} attribute, but {4} names are not accessible to \
+ enterprise bean modules. Switch to a name in one of the following \
+ namespaces: {5}.
+CWWKD1061.comp.name.in.ejb.explanation=The specified namespace is not visible \
+ to the enterprise bean module in which the repository interface is defined.
+CWWKD1061.comp.name.in.ejb.useraction=Update the Repository annotation to \
+ specify a data store name that is visible to the enterprise bean module that \
+ provides the repository interface.
+
+CWWKD1062.resource.not.found=CWWKD1062E: The {0} method of the {1} repository \
+ interface is unable to locate the {2} resource with the {3} filter.
+CWWKD1062.resource.not.found.explanation=The resource is not found.
+CWWKD1062.resource.not.found.useraction=Ensure that the resource and all other \
+ resources upon which it depends exist and are correctly configured.
+
+CWWKD1063.unsupported.resource=CWWKD1063E: The {0} method of the {1} repository \
+ interface cannot obtain a {2} resource because the Jakarta Persistence provider \
+ does not support unwrapping as a {3} class.
```

Comment:
```suggestion
 does not support unwrapping it as a {3} class.
```

Reply:
The proposed change here would result in a meaning that is not correct. I'll need to try writing this different.